CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ## [unreleased] - yyyy-mm-dd
 
+### Added
+* Adding NetworkSimulator tool (#841). Can be used through inspector while in-editor, or through custom in-game UI window in a build.
```

Comment:
Could be worth it in a changelog to add how it is triggered (tab on desktop & 5 finger press on mobile).